.github/workflows/tests.yaml

```diff
@@ -195,10 +195,54 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           files: ./coverage.xml
 
+  tests-qchem:
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.10"]
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+          cache-dependency-path: setup.py
+
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          activate-environment: quacc-env
+
+      - name: Install pip packages
+        run: |
+          pip install .[strict,dev,qchem]
```

Comment:
I added a unique test suite for Q-Chem like I did for dftbplus, psi4, and tblite since they have their own unique dependencies.